include/ui/gtk4.h

```diff
@@ -0,0 +1,156 @@
+#ifndef UI_GTK4_H
```

Comment:
What kind of documentation would you expect? the header is mostly for being able to split the gl area code from the gtk4.c, otherwise a single file could be used

Reply:
I was not referring to document why you need a header, of course. But the documentation of the module, for other developers. What is this, what is its purpose? Something like that. If I look for documentation of a given library that I'm going to use, I will check its header, where you find all the declarations and you do not need to care about the details of the implementation. So it is a good place to document it a bit. One or two lines may do if there is not a lot to say about it.

How's it documented for the current gtk implementation? You can check it for inspiration.

Reply:
Checking at the documentation that the file has now, this line may do:
```
 * Based on the GTK 3 UI backend
```
Or maybe adding a bit more information:
```
 * GTK 4 UI backend based on the GTK 3 backend implementation
```
But your choice :)